### DISCUSSION TITLE
Geräte Firmware für HmIP-BSL lässt sich nicht hochladen

### DISCUSSION THREAD

**Discussion by maik-dmxc** [OpenCCU/OpenCCU#2743] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

das Hochladen von Gerätefirmware wird nicht durchgeführt
Gerätefirmware ist > CCU3 version 

`TypeCode=360
Name=HmIP-BSL
CCUFirmwareVersionMin=3.0.0
CCU3FirmwareVersionMin=3.75.6
FirmwareVersion=2.0.2
`
CCU3 Verson ist
`
Aktuelle Firmwareversion: | 3.75.7.20240420
`

Fehlrmeldung beim Hochladen
`Die Kompatibilät dieses Geräte-Firmware Updates kann nicht überprüft werden. Bitte laden Sie die Firmware erneut herunter und versuchen sie noch einmal.`

### Describe the behavior you expected

Firmwrae muss als korrekt erkannt werden

### Steps to reproduce the issue

1. Firmware auswählen (*.tgz)
2. Firmware hochladen
3. Info Meldung -> Versioncheck nicht möglich
...

### What is the version this bug report is based on?

3.75.7.20240420

### Which base platform are you running?

rpi5 (RaspberryPi5, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
May 13 09:11:17 RaspberryMatic auth.info sshd[30752]: Accepted password for root from 192.168.2.23 port 50491 ssh2
root@RaspberryMatic:/var/log# May 13 08:15:06 RaspberryMatic local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5F298DB0A2:12","LEVEL",0.000000}, result: [faultCode:-
1,faultString:"Generic error"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:15:06 RaspberryMatic local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1008, params: {"001A5F298DB0A2:12","LEVEL",0.000000}) [CallSetValue():iseXmlRpc.cpp:1515]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:15:06 RaspberryMatic local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = 001A5F298DB0A2:12 [WriteValue():iseDOMdpHSS.cpp:76]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:15:25 RaspberryMatic local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5F298DB0A2:12","LEVEL",0.000000}, result: [faultCode:-
1,faultString:"Generic error (TRANSACTION_DISCARDED_FOR_UNREACHABLE_DEVICE)"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:15:25 RaspberryMatic local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1008, params: {"001A5F298DB0A2:12","LEVEL",0.000000}) [CallSetValue():iseXmlRpc.cpp:1515]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:15:25 RaspberryMatic local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = 001A5F298DB0A2:12 [WriteValue():iseDOMdpHSS.cpp:76]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:17:05 RaspberryMatic local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5F298DB0A2:12","LEVEL",0.000000}, result: [faultCode:-
1,faultString:"Generic error"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2618]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:17:05 RaspberryMatic local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1008, params: {"001A5F298DB0A2:12","LEVEL",0.000000}) [CallSetValue():iseXmlRpc.cpp:1515]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 08:17:05 RaspberryMatic local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = 001A5F298DB0A2:12 [WriteValue():iseDOMdpHSS.cpp:76]
-sh: syntax error: unexpected "("
root@RaspberryMatic:/var/log# May 13 09:11:17 RaspberryMatic auth.info sshd[30752]: Accepted password for root from 192.168.2.23 port 50491 ssh2
```

**Comment by jens-maus** [comment, marked as answer]
Das sieht mir nicht nach einem generellen Problem innerhalb von RaspberryMatic aus. Hier wird auch keinerlei solche Probleme mit dem 2.0.2 Update gemeldet:

https://homematic-forum.de/forum/viewtopic.php?f=58&t=82425

Insofern scheint mir das eher ein Nutzungs-/Download Problem zu sein und gehört daher eher in den Diskussionsbereich als in den Bereich wo Bugs/Issue von RaspberryMatic gemeldet werden.

**Comment by Baxxy13** [comment]
Wenn man sich das Logfile so anguckt dann ist da irgendwas im Argen.
001A5F298DB0A2 ist die SN des BSL?

Zwar nicht beliebt, aber ich würde die "eQ-3 Holzhammer Methode" anwenden. 
Also den BSL aus der Zentrale löschen und neu Anlernen.

Das Firmware-File für den BSL `HmIP-BSL_update_V2_0_2_230929.tgz`  konnte ich hier problemlos auf meine Testsysteme "uploaden".
